metals/src/main/scala/scala/meta/internal/metals/Indexer.scala

```diff
@@ -499,7 +500,10 @@ final case class Indexer(
       targetOpt: Option[b.BuildTargetIdentifier],
       data: Seq[TargetData],
   ): Unit = {
-
+    def isExtensionMethod(si: SymbolInformation): Boolean =
+      si.kind == Kind.METHOD && si.properties == 0 && si.symbol(
```

Comment:
Can we move this method to `XtensionSymbolInformation`? Currently `XtensionSymbolInformation` operates on `s.SymbolInformation.Kind` so you might need to create `XtensionSymbolInformationKind`

Reply:
So this is a very hacky method and it works only because of the way ScalaTopLevelMtags are produced, not "in general". So we should definitely not propagate it and I'm trying to come up with a different way to do this.